packages/framework/src/Model/Pricing/Currency/CurrencyFacade.php

```diff
@@ -227,6 +230,24 @@ public function setDomainDefaultCurrency(Currency $currency, $domainId)
         $this->productPriceRecalculationScheduler->scheduleAllProductsForDelayedRecalculation();
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency $originalDefaultCurrency
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency $newDefaultCurrency
+     */
+    protected function recalculateExchangeRatesByNewDefaultCurrency(
+        Currency $originalDefaultCurrency,
+        Currency $newDefaultCurrency
+    ): void {
+        $coefficient = Money::createFromFloat($originalDefaultCurrency->getExchangeRate() / $newDefaultCurrency->getExchangeRate(), 6);
```

Comment:
As in previous commit: `$currency` should not be instance of money.

Besides it is the second occurence of the same calculation. Do you think it could be extracted into new method? Something like `getExchangeRateForCurrencies($inputCurrency, $outputCurrency)`

Reply:
refactored in 277b3c648